lib/MusicBrainz/Server/Data/Editor.pm

```diff
@@ -246,11 +246,22 @@ sub find_subscribers
     $self->query_to_list_limited($query, [$editor_id], $limit, $offset);
 }
 
+sub _die_if_username_invalid {
+    my $name = shift;
+    my $sanitized_name = sanitize($name);
+
+    die 'Invalid user name' if (
+        $name ne $sanitized_name ||
+        $sanitized_name =~ qr{^deleted editor \#\d+$}i
```

Comment:
This is now different in that the deleted editor check is performed on the sanitized version of the name, so I suppose it could ISE with "Invalid user name" if the form validation only checks the name.

Reply:
Wouldn't that get caught by the first (sanitized != name) check anyway at the form level?

Reply:
Yes, that sounds right actually.